packages/compass-e2e-tests/helpers/commands/hide-shell.ts

```diff
@@ -1,12 +1,23 @@
+import { TEST_MULTIPLE_CONNECTIONS } from '../compass';
 import type { CompassBrowser } from '../compass-browser';
 import retryWithBackoff from '../retry-with-backoff';
 import * as Selectors from '../selectors';
 
-export async function hideShell(browser: CompassBrowser): Promise<void> {
-  await retryWithBackoff(async function () {
-    const shellContentElement = await browser.$(Selectors.ShellContent);
-    if (await shellContentElement.isDisplayed()) {
-      await browser.clickVisible(Selectors.ShellExpandButton);
-    }
-  });
+export async function hideShell(
```

Comment:
maybe we can rename this to closeShell (also the file), so that it's easier to find for the future devs